epub33/core/index.html

```diff
@@ -657,15 +657,37 @@ <h4>Introduction</h4>
 						without fallbacks (<a>Core Media Type Resources</a>) and those that cannot (<a>Foreign
 							Resources</a>).</p>
 
-					<p><a>Authors</a> are free to use both types of resources to construct their EPUB Publications, but
-						need to be aware that some Reading Systems might not render the Foreign Resources they use.</p>
-
-					<p>As EPUB Publications are designed to be fully consumable on any compliant Reading System, a
-						system of fallbacks is necessary to ensure that the use of Foreign Resources does not impact on
-						the ability of the user to consume the content. This section lists the <a
-							href="#sec-cmt-supported">set of Core Media Type Resources</a> and identifies <a
-							href="#sec-foreign-restrictions">fallback mechanisms</a> that can be used to satisfy the
-						consumability requirement.</p>
+					<p>Formats are typically only included as Core Media Type Resources when it can be shown that they
+						have broad support in web browser cores &#8212; the rendering engines on which EPUB 3 Reading
+						Systems are built. They are like a compact between <a>Authors</a> and Reading Systems to ensure
+						the predictability of rendering of EPUB Publications.</p>
```

Comment:
compact -> contract

Reply:
Compact: an agreement or covenant between two or more parties
Contract: a binding agreement between two or more persons or parties, especially : one legally enforceable

Contract isn't really the right word for this.

Reply:
But since this seems to confuse people, maybe sticking to agreement is simplest.

Reply:
Now reads:

> They are an agreement between Reading System developers and Authors to ensure the predictability of rendering of EPUB Publications.

Reply:
I must admit I have never met the term 'compact' in that meaning, although you are right, it is even on https://www.thefreedictionary.com/compact ...